src/lib/holocene/month-picker.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { monthNames } from '$lib/utilities/calendar';
+
+  export let months: string[];
+
+  const onClick = (e: MouseEvent, month: string) => {
+    if (e.metaKey && month !== '*') {
+      if (months.includes('*')) {
+        months = months.filter((m) => m !== '*');
+      }
+      if (months.includes(month)) {
+        months = months.filter((m) => m !== month);
+      } else {
+        months = [...months, month];
+      }
+    } else {
+      months = [month];
+    }
+  };
+</script>
+
+<div class="flex flex-wrap gap-4 text-center">
+  {#each monthNames as { label, value }}
+    <button
+      class="cell"
```

Comment:
Seen a few of these `<button class="cell" />` maybe it warrants its own component?

Reply:
These are so specific to these month/day pickers, rather just leave them in here.